src/Experimental/isomorphism.jl

```diff
@@ -173,9 +171,40 @@ function has_isomorph(
     vertex_relation::Union{Nothing,Function}=nothing,
     edge_relation::Union{Nothing,Function}=nothing,
 )::Bool
-    return has_isomorph(
-        g1, g2, alg; vertex_relation=vertex_relation, edge_relation=edge_relation
-    )
+    throw(MethodError(has_isomorph, (g1, g2, alg)))
+end
+
+"""
+    canonize!(g, alg::IsomorphismAlgorithm=AlgNautyGraphs())
+
+Permute the vertices of graph `g` into the canonical order defined by the algorithm `alg` and return the permutation. 
+If graphs `g1` and `g2` are isomorphic, the orders of their vertices will be equal after canonizing them with the same algorithm.
+
+### Optional Arguments
+- `alg`: The algorithm that is used to canonize the graph. Can be only be `AlgNautyGraphs()`
+   at this moment, which requires `NautyGraphs` to be installed and imported.
+
+### Examples
+```doctest.jl
```

Comment:
huh... I did not know that you can also use `doctest.jl` as a specifier for doctest!